src/pages/burgerdetail/BurgerDetailPage.tsx

```diff
@@ -0,0 +1,156 @@
+import { useState } from 'react';
+import * as S from './BurgerDetailPage.style';
+import { Ellipse, Cimg11, IcMinus, IcPlus } from '@assets/svgs';
+
+const BurgerDetailPage = () => {
+  const [activeIndex, setActiveIndex] = useState<number[]>([]);
+
+  const toggle = (index: number) => {
+    if (activeIndex.includes(index)) {
+      // 이미 열려있으면 닫기
+      setActiveIndex(activeIndex.filter((i) => i !== index));
+    } else {
+      // 닫혀있으면 열기
+      setActiveIndex([...activeIndex, index]);
+    }
+  };
```

Comment:
천재다!!!!!!!!!!!!!!!!!!!!

Reply:
천재만재 개발자

Reply:
그런데 지금 activeIndex로 값을 접근하고 있는데 set함수에서는 prev로 접근하는게 더 안전합니다!!
```suggestion
  const toggle = (index: number) => {
    setActiveIndex((prev) =>
    prev.includes(index)
      ? prev.filter((i) => i !== index) // 이미 열려있으면 닫기
      : [...prev, index] // 닫혀있으면 열기
   );
  };
```

Reply:
우왕.... 대박이다 진짜
고맙ㄷ쁘니당